planning/scenario_planning/parking/costmap_generator/package.xml

```diff
@@ -21,6 +21,8 @@
   <depend>tf2</depend>
   <depend>tf2_ros</depend>
   <depend>lanelet2_extension</depend>
+  <depend>orocos_kdl</depend>
+  <depend>pcl</depend>
```

Comment:
again, we shouldn't need orocos_kdl package.
Als, the key for pcl should be `libpcl-all-dev`
```suggestion
  <depend>libpcl-all-dev</depend>
```

Reply:
This is the error I am getting:

```
/usr/bin/ld: CMakeFiles/costmap_generator.dir/nodes/costmap_generator/costmap_generator_node.cpp.o: in function `KDL::operator*(KDL::Frame const&, KDL::Frame const&)':
costmap_generator_node.cpp:(.text._ZN3KDLmlERKNS_5FrameES2_[_ZN3KDLmlERKNS_5FrameES2_]+0x99): undefined reference to `KDL::operator*(KDL::Rotation const&, KDL::Rotation const&)'
/usr/bin/ld: CMakeFiles/costmap_generator.dir/nodes/costmap_generator/costmap_generator_node.cpp.o: in function `tf2::gmTransformToKDL(geometry_msgs::msg::TransformStamped_<std::allocator<void> > const&)':
costmap_generator_node.cpp:(.text._ZN3tf216gmTransformToKDLERKN13geometry_msgs3msg17TransformStamped_ISaIvEEE[_ZN3tf216gmTransformToKDLERKN13geometry_msgs3msg17TransformStamped_ISaIvEEE]+0x92): undefined reference to `KDL::Rotation::Quaternion(double, double, double, double)'
/usr/bin/ld: CMakeFiles/costmap_generator.dir/nodes/costmap_generator/costmap_generator_node.cpp.o: in function `void tf2::doTransform<geometry_msgs::msg::PoseStamped_<std::allocator<void> > >(geometry_msgs::msg::PoseStamped_<std::allocator<void> > const&, geometry_msgs::msg::PoseStamped_<std::allocator<void> >&, geometry_msgs::msg::TransformStamped_<std::allocator<void> > const&)':
costmap_generator_node.cpp:(.text._ZN3tf211doTransformIN13geometry_msgs3msg12PoseStamped_ISaIvEEEEEvRKT_RS6_RKNS2_17TransformStamped_IS4_EE[_ZN3tf211doTransformIN13geometry_msgs3msg12PoseStamped_ISaIvEEEEEvRKT_RS6_RKNS2_17TransformStamped_IS4_EE]+0xbd): undefined reference to `KDL::Rotation::Quaternion(double, double, double, double)'
/usr/bin/ld: costmap_generator_node.cpp:(.text._ZN3tf211doTransformIN13geometry_msgs3msg12PoseStamped_ISaIvEEEEEvRKT_RS6_RKNS2_17TransformStamped_IS4_EE[_ZN3tf211doTransformIN13geometry_msgs3msg12PoseStamped_ISaIvEEEEEvRKT_RS6_RKNS2_17TransformStamped_IS4_EE]+0x1be): undefined reference to `KDL::Rotation::GetQuaternion(double&, double&, double&, double&) const'
```

Reply:
We had a call an fixed this issue. It was missing dependency to tf2_geometry_msgs in the package.xml

Reply:
@mitsudome-r this is fixed